src/controller/CHIPDeviceController.cpp

```diff
@@ -2284,6 +2393,91 @@ void DeviceCommissioner::PerformCommissioningStep(DeviceProxy * proxy, Commissio
         mReadClient     = std::move(readClient);
     }
     break;
+    case CommissioningStage::kReadTimeSyncInfo: {
+        // This is done in a separate step since we've already used up all the available read paths in the previous read step
```

Comment:
I think we should move this into the main read step, since it always has to happen, and move the optional Fabrics attribute read, which not all commissioners need to do, into the separate step.  That way we have the same number of round trips for most consumers, except the ones that ask for the Fabrics read.

Reply:
kk, this is now changed.